capstone/src/test/java/com/google/sps/clubs/ClubServletTest.java

```diff
@@ -65,70 +69,114 @@ public void setUp() throws IOException {
     this.blobstoreServlet = new BlobstoreServlet();
     datastore = DatastoreServiceFactory.getDatastoreService();
     blobstore = Mockito.mock(BlobstoreService.class);
-  }
-
-  @After
-  public void tearDown() {
-    helper.tearDown();
-  }
 
-  @Test
-  public void doPost_registerNewValidClub() throws ServletException, IOException {
     helper.setEnvEmail(TEST_EMAIL).setEnvAuthDomain("google.com").setEnvIsLoggedIn(true);
-    when(request.getParameter(Constants.CLUB_NAME_PROP)).thenReturn(SAMPLE_CLUB_NAME);
+    when(request.getParameter(Constants.PROPERTY_NAME)).thenReturn(SAMPLE_CLUB_NAME);
     when(request.getParameter(Constants.DESCRIP_PROP)).thenReturn(SAMPLE_CLUB_DESC_1);
     when(request.getParameter(Constants.WEBSITE_PROP)).thenReturn(SAMPLE_CLUB_WEB);
 
     ImmutableList<BlobKey> keys = ImmutableList.of(SAMPLE_BLOB);
     ImmutableMap<String, List<BlobKey>> blobMap = ImmutableMap.of(Constants.LOGO_PROP, keys);
     when(blobstore.getUploads(request)).thenReturn(blobMap);
+  }
```

Comment:
It looks like this is part of setUp now? It would be preferable for this to be defined in the individual test since "when" is used conditionally. If these exact same conditions are used in the multiple tests, you can define a helper method for these and then call them in each test.